service/controller/key/common.go

```diff
@@ -180,6 +183,11 @@ func IsDeleted(getter DeletionTimestampGetter) bool {
 	return getter.GetDeletionTimestamp() != nil
 }
 
+func IsV19Release(releaseVersion *semver.Version) bool {
```

Comment:
How about checking the major version is v19 or higher? I'm afraid comparison of patch versions might be tricky and unreliable

Reply:
Agree, I slightly changed it to `return releaseVersion.Major >= v19.Major`. Is this better for you?

Reply:
Yes perfect